src/analysis/normed_space/operator_norm.lean

```diff
@@ -353,6 +353,85 @@ begin
       by rwa [mul_one] }
 end
 
+section uniformly_extend
+
+variables [complete_space F] {e : E →L[𝕜] G} (h_dense : dense_range e)
+
+section
+variables (h_e : uniform_inducing e)
+
+/-- Extension of a continuous linear map `f : E →L[𝕜] F`, with `E` a normed space and `F` a complete
+    normed space, along a uniform and dense embedding `e : E →L[𝕜] G`.  -/
+def extend : G →L[𝕜] F :=
+/- extension of `f` is continuous -/
+have cont : _ := (uniform_continuous_uniformly_extend h_e h_dense f.uniform_continuous).continuous,
+/- extension of `f` agrees with `f` on the domain of the embedding `e` -/
+have eq : _ := uniformly_extend_of_ind h_e h_dense f.uniform_continuous,
+{ to_fun := (h_e.dense_inducing h_dense).extend f,
+  add :=
+  begin
+    refine is_closed_property2 h_dense (is_closed_eq _ _) _,
+    { exact cont.comp (_root_.continuous_add continuous_fst continuous_snd) },
+    { exact _root_.continuous_add (cont.comp continuous_fst) (cont.comp continuous_snd) },
+    { assume x y, rw ← e.map_add, simp only [eq], exact f.map_add _ _  },
+  end,
+  smul := λk,
+  begin
+    refine is_closed_property h_dense (is_closed_eq _ _) _,
+    { exact cont.comp (continuous_smul continuous_const continuous_id)  },
+    { exact (continuous_smul continuous_const continuous_id).comp cont },
+    { assume x, rw ← map_smul, simp only [eq], exact map_smul _ _ _  },
+  end,
+  cont := cont
+}
+
+lemma extend_zero : extend (0 : E →L[𝕜] F) h_dense h_e = 0 :=
+begin
+  apply ext,
+  refine is_closed_property h_dense (is_closed_eq _ _) _,
+  { exact (uniform_continuous_uniformly_extend h_e h_dense uniform_continuous_const).continuous },
+  { simp only [zero_apply], exact continuous_const },
+  { assume x, exact uniformly_extend_of_ind h_e h_dense uniform_continuous_const x }
+end
+
+end
+
+section
+variables {N : ℝ} (h_e : ∀x, ∥x∥ ≤ N * ∥e x∥)
+
+local notation `ψ` := f.extend h_dense (uniform_embedding_of_bound _ _ h_e).to_uniform_inducing
+
+/-- If a dense embedding `e : E →L[𝕜] G` expands the norm by a constant factor `N⁻¹`, and the norm
+    of `f : E →L[𝕜] G` is bounded by `M`, then the norm of the extension of `f` along `e` is bounded
+    by `M * N`. -/
+lemma op_norm_uniformly_extend_le : ∥ψ∥ ≤ N * ∥f∥ :=
```

Comment:
I would remove the `uniformly` in the name, as your function is called `extend`.